### DISCUSSION TITLE
What are the features of your digital clock project?

### DISCUSSION THREAD

**Discussion by shivay-coding** [Ajay-Dhangar/digital-clock#2] [Q&A] [answered]
![image](https://github.com/Ajay-Dhangar/digital-clock/assets/101887417/210141d2-04d4-4fdd-abbd-9a252fbb6b47)

**Comment by ajay-dhangar** [comment, marked as answer]
A: The digital clock project provides the following features:

- Real-time display: It shows the current time accurately.
- 12-hour and 24-hour formats: Users can choose between the standard 12-hour format or the 24-hour format.
- AM/PM indicator: In the 12-hour format, it indicates whether it is morning or afternoon.
- User-friendly interface: The clock has a clean and intuitive design, making it easy to read and interact with.
